### DISCUSSION TITLE
Feature Request: Bulk conversion of draft issues to issues

### DISCUSSION THREAD

**Discussion by jefflbrauer** [community/community#61687] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

As a project manager utilizing project templates with multiple draft issues, I would like to bulk convert draft issues to issues after creating a project from a template so that I save time and effort not having to manually convert each task for every project. 

**Acceptance Criteria:**
- The table view for a project allows select all (issues) capability
- Projects allows for bulk editing of selected issues
- Bulk Edit capability includes "Convert to Issue" action

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by BenKellySoftware** [comment]
Ideally, there should be an option to allow issues created in a project board to automatically be added to that project instead of starting as drafts.

**Comment by brhoff720** [comment]
While up voted the original issue, want to say that this would drastically improve my workflow.  A project template with 70+ draft issues, it really is painful to convert each individual draft issue to issue and assign to the same repo.   I generally have 1 to 4 new projects a month that follow this workflow.

**Comment by HeatPhoenix** [comment]
Upvoted, the UI is currently super unwieldy and I have to convert them manually one by one.

**Comment by siddpuri** [comment]
When choosing between GitHub issues vs some other task management system, this ends up being the decisive missing feature for me.

**Comment by blaberry** [comment]
This is probably one of the biggest pain points, especially when converting from any other previous system. We had over 100 issues to convert and the current flow was egregiously painful to the point that we had to stop and think about whether the benefits outweighed the clunkiness.

**Comment by MakePixelsWork** [comment]
Maybe this should be available for tasks lists too. Now I create tasks for something, and if I want to convert them to issues, I have to do them one by one.

I can imagine that you might not want all of them to be issues, so converting them all to issues, maybe needs a checklist dialog, where you can first select a Convert all option, then get a dialog with checkboxes and then can de-select those you do not want to convert.

Example of dropdown menu, where to option can be added.
![image](https://github.com/community/community/assets/56885246/0102b847-ce9a-4fa6-8347-e9295dbef8be)

**Comment by DrewUniverse** [comment]
As someone finding it necessary to leave Trello, Github is mostly great but lacking a few little things like this. I wish I could rapid-fire input a ton of issues, but I have to do them as drafts and then manually convert them all one at a time? It takes so, so long for projects that necessitate hundreds and hundreds of entries. Send help, please!

**Comment by pbower** [comment]
Lack of this feature is HELL

**Comment by falarosee** [comment]
We need this too please

**Comment by iangmaia** [comment]
I wrote simple a Ruby script to automate this (it can be tweaked to fit more sophisticated filters or workflows): https://gist.github.com/iangmaia/6d75d796531f46fd4d4b49bcc6070873

**Comment by raulk** [comment]
Here's a workaround I discovered today that is entirely keyboard driven. In a Table view:

1. Add the Repository column to the view.
2. Set the `is:draft no:repo` filter for the view so that only drafts with no repo are visible.
3. Select the Repository cell for the first issue.
4. Press Enter and type the repo name (you can copy it to clipboard for later).
5. Press Enter to select that repo. This creates an issue there and moves the cursor down to the next row.
6. Keep pressing Enter, pasting the repo name, and pressing Enter again.

It's kinda cumbersome, but much faster than pointing and clicking. Maybe this will be solved for real one day; doesn't appear too complicated from the outside.

**Comment by albertkun** [comment]
Would also love to see this feature implemented!!

**Comment by ahad-arain** [comment]
+1 for this feature request for Project Management.

**Comment by mikem-of** [comment]
+1 plz
